### ISSUE TITLE
Widget Layout - Hide time "All-day"

### ISSUE THREAD

**Issue by Andarin** [SimpleMobileTools/Simple-Calendar#1624]
**Tl;dr: In the widget, would it be possible to hide "All-day" as a time ?**

<img src="https://user-images.githubusercontent.com/1506838/151595666-fcbd89c9-6413-4148-8a92-416cc456b856.png" width="75%" height="75%">

Use case:

I'm using the Simple Calendar, especially the widget. I have a calendar dedicated to birthdays, which are always "All-day". In order to make more out of my screen space, it would be great to not have a line "All-day" below every birthday. 
Would it be possible to add an option to note have this line ? (exactly like events without location don't have a location line).

Alternatively, individual formatting for different calendars in the widget would also do the trick, but it would be a lot less "simple" in UX and implementation.

Thank you for these great apps, the nice layout of the widget and the possibility to modify all colors.

**Comment by tibbi** [comment]
dont really expect having it for now, sounds quite confusing

**Comment by v4hn** [comment]
I would like to support this request one way or another. This is my current widget in use: ![Screenshot_20221115-091306_1_1.png](https://user-images.githubusercontent.com/680358/201867826-eac2a553-87b6-425a-ab66-7a286b477dbe.png)

As you can see, most of the space is used to tell me someone is on vacation or show some other marginal all-day info.
I would actually prefer to be able to disable specific categories (not calendars) in the widget altogether without disabling them in the app, but *anything* that can be done to safe space with all day events would help.
For example I don't see why the widget would need to show the start date of multi-day all-day events either.

Another simple solution to handle all-day events would be to move them to the bottom of the list so that the fine-grained day-plan shows by default.

**Comment by tibbi** [comment]
some redesigns are on the roadmap
